_layouts/home.html

```diff
@@ -23,78 +23,87 @@
   <div class="container">
     <div class="row justify-content-start">
       <div class="col-12 col-md-7 col-lg-5 col-md-3 order-2 order-md-2" id="content">
-        <h1>2023兔年<br>“载歌在谷”春晚<br>赞助商招募火热进行中</h1>
+        <h2>2023兔年<br>“载歌在谷”春晚<br>赞助商招募火热进行中</h2>
         <p>
-            兔年春晚及游园时间：2023年1月21日 <br>
-            地点：San Jose Center for Performing Arts <br>
-            联系赞助组 Email: sponsor-gala@zgzg.io <br>
-            WeChat: “载歌在谷” <br>
-            招商类别与赞助商权益详情，请扫描二维码
+            <b>兔年春晚及游园时间：</b>2023年1月21日 <br>
+            <b>地点：</b>San Jose Center for Performing Arts <br>
+            <b>联系赞助组 Email: </b><u>sponsor-gala@zgzg.io</u> <br>
+            <b>WeChat公众号:</b> “载歌在谷” <br>
+            招商类别与赞助商权益详情，请访问以下链接
         </p>
-        <img src="https://tva1.sinaimg.cn/large/008vxvgGgy1h7qq9nha31j306y06z74o.jpg" alt="报名二维码" />
+        <a href="https://mp.weixin.qq.com/s/BzpzveTu9Uyzw9j3c1Nmpg">
+          <button type="button" class="btn btn-light btn-primary btn-lg">查看详情</button>
+        </a>
       </div>
     </div>
   </div>
 </div>
 
-<!-- Tab links -->
-<div class="tab">
-  <button class="tab-links default-open" onclick="openTab(event)" data-tab-group="main" data-tab-id="programmes_chunwan">载歌在谷精彩活动</button>
-  <button class="tab-links" onclick="openTab(event)" data-tab-group="main" data-tab-id="programmes">“云集”系列栏目</button>
-  <button class="tab-links" onclick="openTab(event)" data-tab-group="main" data-tab-id="sponsor">赞助商</button>
-</div>
+<div class="strip">
+  <div class="container">
 
-<!-- Tab content -->
-<div id="programmes_chunwan" class="tab-content">
-  <div class="strip">
-    <div class="container pt-6 pb-6 pt-md-6 pb-md-6">
-      <h1>载歌在谷精彩活动</h1>
-      <div class="row justify-content-start">
-        {% if site.data.data.programmes_chunwan %}
-          {% for item in site.data.data.programmes_chunwan %}
-            {% assign remainder = forloop.index0 | modulo: 2 %}
-            {% if remainder == 0 %}
-              {% include list-item-left.html %}
-            {% else %}
-              {% include list-item-right.html %}
-            {% endif %}
-          {% endfor %}
-        {% endif %}
+      <div class="home-section-header pb-large">
+        <h2>关于 “载歌在谷”</h2>
+        <div class="divider">
+          <hr></hr>
+        </div>
+        <div class="home-section-content">
+          <p class="py-2">“载歌在谷”社区是一个以硅谷高科技华人从业者为主要贡献者的非盈利组织。“载歌在谷”社区旨在丰富硅谷华人的文化生活、弘扬中华文化。社区以文艺活动为主要载体，如“载歌在谷”春节晚会。</p>
+        </div>
       </div>
-    </div>
+
+      {% if site.data.data.programmes_chunwan %}
+        {% for item in site.data.data.programmes_chunwan %}
+          {% assign remainder = forloop.index0 | modulo: 2 %}
+          {% if remainder == 0 %}
+            {% include list-item-left.html %}
```

Comment:
GENIUS! That's how it should be done!

Reply:
This was taken from Wenyuan's change :) Agree that this is neat!